Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -117,28 +117,49 @@ defineSuite([
         });
     });
 
+    it('rejects readyPromise with invalid tileset version', function() {
+        var tilesJson = {
+            "asset" : {
+                "version" : "2.0"
+            }
```

Comment:
Since it is stringified, it might be cleaner to remove the quotation marks. Doesn't really matter though.